src/Qir/Common/cmake/unit_test_include.cmake

```diff
@@ -15,9 +13,10 @@ macro(add_unit_test target)
     endif()
 
     set(TEST_DEPS2 "${CMAKE_BINARY_DIR}/bin")
+    set(TEST_DEPS3 "${PROJECT_SOURCE_DIR}/../Runtime/build/${CMAKE_BUILD_TYPE}/bin")
     set_property(TEST ${target} PROPERTY ENVIRONMENT
-        "LD_LIBRARY_PATH=${TEST_DEPS1}:${TEST_DEPS2}:${LD_LIBRARY_PATH}"
-        "PATH=${TEST_DEPS1}\;${TEST_DEPS2}\;${PATH}"
-        "DYLD_LIBRARY_PATH=${TEST_DEPS1}:${TEST_DEPS2}:${DYLD_LIBRARY_PATH}"
+        "LD_LIBRARY_PATH=${TEST_DEPS1}:${TEST_DEPS2}:${TEST_DEPS3}:${LD_LIBRARY_PATH}"
+        "PATH=${TEST_DEPS1}\;${TEST_DEPS2}\;${TEST_DEPS3}\;${PATH}"
+        "DYLD_LIBRARY_PATH=${TEST_DEPS1}:${TEST_DEPS2}:${TEST_DEPS3}:${DYLD_LIBRARY_PATH}"
```

Comment:
(Probably something to figure out later (in a separate issue maybe?), it was existing before this change)
I'm rather surprised to see in a single block the env var `LD_LIBRARY_PATH` (typical for UNIX/Linux), and the `;` as a separator in the env var `PATH` (typical for Win but I have never seen `;` as a separator in UNIX/Linux, `:` only)

Reply:
yeah, this is a little odd but appears to just be a shortcut. It sets all three environment variables to avoid having an if-statement that sets only the variable for the current platform. We could clean this up later to avoid confusion, but it would require making the code here longer.

Reply:
FYI: #601 Double-check that the PATH is set correctly